### PR TITLE
feat(edgeless): support lasso tool

### PR DESCRIPTION
**Pull request by golok727**
Drag to create a new selection
`Shift` + Drag to add to the selection
`Alt` + Drag to subtract from the selection
image

Closes https://github.com/toeverything/blocksuite/issues/6363

### CODE REVIEW

**Comment by golok727** [comment]
@doodlewind can I change the shortcut of Connector to `C` and the lasso tool to `L`?

File: tests/edgeless/shortcut.spec.ts [hunk 1]
```diff
@@ -57,13 +57,36 @@ test('shortcut', async ({ page }) => {
...
+  const lassoButton = locatorEdgelessToolButton(page, 'lasso');
+  await expect(lassoButton).toHaveAttribute('active', '');
+});
+
+test('toggle lasso tool modes', async ({ page }) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyEdgelessState(page);
+  await switchEditorMode(page);
   await page.mouse.click(100, 100);
-  await page.keyboard.press('x');
```
Comment (golok727) [inline tests/edgeless/shortcut.spec.ts:65]: This shortcut was removed in https://github.com/toeverything/blocksuite/pull/6573

**Comment by golok727** [comment]
@doodlewind PTAL

**Review by doouding — APPROVED** [review]
Thanks for the feature : )

**Pull request merged**